quality-metrics/qm-pull-requests.md

```diff
@@ -0,0 +1,25 @@
+---
+Author: John Bellone <jbellone@bloomberg>
+Status: Draft
+License: Apache 2.0
+---
+
+## Cookbook has closed pull-requests on GitHub.
```

Comment:
The challenge here is that we don't focus directly on Github. We also have Bitbucket, hell even some public GitLab instances.